### PR TITLE
Make a few scheduler errors less critical.

### PR DESCRIPTION
**Pull request by mstratford**
Lowers the number of emails Computing gets for common user mistakes.

### CODE REVIEW

File: src/Classes/ServiceAPI/MyRadio_Season.php [hunk 1]
```diff
@@ -888,20 +888,23 @@ public function schedule($params)
...
+            throw new MyRadioException(
+                'No valid Time was sent to the Scheduling Mapper.',
+                400
+            );
         }
         if ($params['time'] != -1 && !isset($this->requested_times[$params['time']])) {
             throw new MyRadioException(
                 'The Time value sent is not a valid Requested Time Reference.',
-                MyRadioException::FATAL
+                500
```
Comment (LordAro) [inline src/Classes/ServiceAPI/MyRadio_Season.php]: Why is this one a 500?

Pretty sure none of these exceptions need to be split over multiple lines either

Comment (mstratford) [reply src/Classes/ServiceAPI/MyRadio_Season.php]: The multiple lines thing was due to 120 char line length.
Could be a 400, wasn't sure given it was once a "FATAL" error.

Comment (LordAro) [reply src/Classes/ServiceAPI/MyRadio_Season.php]: All the others were FATAL as well :p
The only line that's gotten longer is the last one, all the otherse are shorter than they were

Comment (mstratford) [reply src/Classes/ServiceAPI/MyRadio_Season.php]: I think the issue was it was ignoring the lengths, until I changed them and they were still over 120chars. I've changed the 500 now though.

**Pull request merged**